docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

```diff
@@ -41,8 +44,26 @@ java.util.logging.ConsoleHandler.level=ALL
 software.amazon.jdbc.Driver.level=FINER
 software.amazon.jdbc.plugin.level=FINER
 ```
-
+#### Connection Parameter
 The AWS JDBC Driver also has a parameter, [`wrapperLoggerLevel`](#aws-advanced-jdbc-driver-parameters), to configure the logging level.
+```java
+ final Properties properties = new Properties();
+ properties.setProperty("wrapperLoggerLevel", "finest");
+ Connection conn = DriverManager.getConnection(CONNECTION_STRING, properties);
+```
+
+The two methods above are more suitable for simple Java applications that do not have a 3rd party logging library configured. If you are using the AWS JDBC Driver with Spring or Spring Boot, you can enable logging by adding the following to your configuration file.
+For `application.yml`:
+```yaml
+logging:  
+  level:  
+    software.amazon.jdbc: TRACE
+```
+
+For `application.properties`:
+```properties
+logging.level.software.amazon.jdbc=trace
+```
```

Comment:
Maybe have this moved to the logging section instead, so it doesn't look like a part of the connection parameter block?